roles/aes-decrypt/tasks/main.yml

```diff
@@ -1,23 +1,26 @@
 ---
-- name: Create a temporary file for ciphertext
-  tempfile:
-    state: file
-  register: ciphertext_temp
-- name: Copy ciphertext to temp file
-  copy:
-    src: "{{ ciphertext }}"
-    dest: "{{ ciphertext_temp.path }}"
-- name: Truncate temporary ciphertext path
-  shell: "truncate --size=-16 '{{ ciphertext_temp.path }}'"
-- name: Read IV
-  shell: tail -c 16 '{{ ciphertext }}' | xxd -p -c 32
-  register: iv
-- name: Decrypt data
-  shell: |
-    openssl enc \
-      -aes-256-cbc \
-      -d \
-      -in "{{ ciphertext_temp.path }}" \
-      -out "{{ plaintext }}" \
-      -K "{{ key_hex }}" \
-      -iv "{{ iv.stdout }}"
+- block:
```

Comment:
Do roles need to have the tags?

Or the same behaviour also applies to `include_role`?

Reply:
I tried to revert all `tasks/main.yml` in roles, it doesn't work for 2.5. I guess the files here are considered just like tasks.

Reply:
So `import|include_role` simply inlines the tasks as you observed. See https://docs.ansible.com/ansible/devel/user_guide/playbooks_reuse.html

Wonder if we can get around this with static imports instead. 

Why must Ansible developers make the behaviour so strange and complicated? >_>